github/REVIEWING.md

```diff
@@ -0,0 +1,341 @@
+# Git / GitHub  Reviewing Workflow
+
+Reviewing is done via the GitHub pull request interface, generally following the [GitHub guidelines](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests).
+
+Reviewing implies 5 actions:
+
+1. Reviewing the commit messages and metadata (signature, authorship)
```

Comment:
```suggestion
1. Reviewing the commit messages and metadata (signature, authorship, CLA/DCO)
```

Reply:
All of these are requirements that vary from repo to repo, not all are mandatory and absence of some means the review process is simplified.